src/vscode-bicep/schemas/bicepconfig.schema.json

```diff
@@ -237,14 +237,33 @@
           "description": "Template Spec module alias definitions",
           "type": "object",
           "additionalProperties": {
-            "$ref": "#/definitions/templateSpecModuleAlias"
+            "$ref": "#/definitions/templateSpecModuleAlias",
+            "additionalProperties": false
           }
         },
         "br": {
           "title": "Bicep Registry Module Aliases",
           "description": "Bicep Registry module alias definitions",
           "additionalProperties": {
-            "$ref": "#/definitions/bicepRegistryModuleAlias"
+            "$ref": "#/definitions/bicepRegistryProviderAlias",
```

Comment:
Should this should still be pointing to `#/definitions/bicepRegistryModuleAlias` since it's within `moduleAliases`?